index.html

```diff
@@ -26,5 +27,121 @@ <h2 class="intro-section-item" id="intro-text2">I’am a Software Developer</h2>
       <li><a href="" class="icon-element" type="button"><img src="src/m.png" alt="m"></a></li>
     </ul>
   </section>
+
+  <section class="recent-works">
+    <h1 class="works">My Recent Works</h1>
+    <img class="works-line" src="src/line.png" alt="line">
+
+    <div class="project-container project1">
+      <img class="project-item project-image" src="src/project-image.png" alt="project-image">
+      <h1 class="project-item project-name">Multi-Post Stories</h1>
+      <p class="project-item project-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="project-item lang-container">
+        <li class="lang-item lang-css">css</li>
+        <li class="lang-item lang-html">html</li>
+        <li class="lang-item lang-bootstrap">bootstrap</li>
+        <li class="lang-item lang-ruby">Ruby</li>
+      </ul>
+      <button type="button" class="project-item project-button buttons">See Project</button> 
+    </div>
+
+    <section class="card-grid-container">
```

Comment:
I suggest you fix the margins for this section to match the figma design. Your design looks great but its a requirement to pass this milestone
<img width="1669" alt="Screenshot 2022-10-28 at 23 34 59" src="https://user-images.githubusercontent.com/48546992/198727615-67f37a6c-58a4-41e2-86f5-f3183158787b.png">

this is your design
<img width="1662" alt="Screenshot 2022-10-28 at 23 31 13" src="https://user-images.githubusercontent.com/48546992/198727676-d9db57cf-ccb8-4b25-9c8c-c59adffd1a10.png">

this is how it should look like
<img width="388" alt="Screenshot 2022-10-28 at 23 31 06" src="https://user-images.githubusercontent.com/48546992/198727733-f8d583ab-0906-44ae-be6b-a5c493c5c6ab.png">

Reply:
heya, regarding a comment on pr about margin. The page is designed for mobile with a width of 375px or less. so anything more than this will result in an increase in margin. Please use a mobile view or I am attaching a screenshot from my mobile from the GitHub page which is using this specific branch that is to be merged.

Reply:
@Nyame-Wolf 
![IMG_0354](https://user-images.githubusercontent.com/43726580/198730467-f77a04c4-f41d-4e2a-9021-37e72ccff101.PNG)

Reply:
@Anand8317 I have just checked it and it looks awesome in 375px. I will put this as an optional comment so that you may or may not change it. For mobile, in a later milestone you need to handle devices starting from 767px. For this milestone however, you have met the requirements.
You have really good work. Continue rocking.